hack/ci/e2e.sh

```diff
@@ -106,9 +106,16 @@ apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
 # the control plane node
 - role: control-plane
+  extraMounts:
+  - containerPath: /etc/audit-policy.yaml
+  - hostPath: ${ARTIFACTS}/audit-policy.yaml
 - role: worker
 - role: worker
 EOF
+    # create the audit-policy
+    wget https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/gce/gci/configure-helper.sh
```

Comment:
Based on https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy (towards tho bottom it references the ~configure-helper.sh~

I thought about sourcing it directly, OR we could generate one and embed it.